### PR TITLE
Added basic browser tests using Cypress

### PR DESCRIPTION
**Pull request by evrifaessa**
This PR adds some basic browser tests using [Cypress](https://www.cypress.io/).

Cypress is a great framework to create browser tests. It's pretty easy to use (in fact it's my first time using it and it was a great experience), it runs tests pretty quickly even on my fairly low-spec laptop. On top of UI tests, it is also possible to easily test API endpoints. I've added a couple of API tests as well.

It also features a great dashboard and a "Studio" feature allowing you to write tests even faster using visual tools, it's very easy for any team to adopt.

This particular PR won't implement it as a workflow or anything. I just wanted to commit the files and get your ideas before going any further.

I would definitely recommend you to check their website out and get a little used to it, because in order to ensure its continuity in the long term, we all will need to help write tests. I'm pretty sure you will like it.

### CODE REVIEW

File: src/composer.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "name": "fossbilling/fossbilling",
     "description": "Free and Open Source Billing System",
-    "version": "0.2.10",
```
Comment (jaapmarcus) [inline src/composer.json]: This should remain 0.2.10

Comment (evrifaessa) [reply src/composer.json]: It was "4.22.0-beta.5" before the PR, updating it without regenerating the lock file throws an error. I don't have a PHP 8.x installation locally so can't really do it.

**Comment by John-S4** [comment]
Cypress looks really interesting. I hadn't even heard of it until today, and haven't had a chance to play with it at all but it definitely looks good and has a really nice UI (which is unusual for anything test related). 

I'm not against at all, if people are going to actually make use of it.

**Review by jaapmarcus — APPROVED** [review]

File: package.json [hunk 2]
```diff
@@ -1,9 +1,14 @@
 {
   "name": "fossbilling",
-  "version": "0.1.0",
+  "version": "0.2.10",
```
Comment (BelleNottelling) [inline package.json:3]: Can we remove the package ver from here to?
No reason in having it, and it's just another thing that has to be periodically changed

**Review by BelleNottelling — APPROVED** [review]
Happy to merge once the conflict is resolved

**Pull request merged**